index.html

```diff
@@ -0,0 +1,110 @@
+        </head>
+
+        <body>
+        <div class="app">
+                <nav class="nav-bar">
+                    <ul class="nav-bar-list">
+                        <li><a class="nav-link-text" href="#">
+                            <img src="images/landify-logo.png" alt="landify's logo">
+                        </a></li>
+                        <li><a class="nav-link-text" href="#">About</a></li>
+                        <li><a class="nav-link-text" href="#">Products</a></li>
+                        <li><a class="nav-link-text" href="#">Pricing</a></li>
+                        <li><a class="nav-link-text" href="#">Blog</a></li>
+                        <li><a class="nav-link-text" href="#">Jobs</a></li>
+                        <li><a class="nav-link-text" href="#">More</a></li>
+                    </ul>
+                    <ul class="nav-bar-list-right">
+                        <li><a class="nav-link-text" href="#">
+                            <img src="images/googleplay-badge.png" alt="Google Play Badge">
+                        </a></li>
+                        <li><a class="nav-link-text" href="#">
+                            <img src="images/appstore-badge.png" alt="App store Badge">
+                        </a></li>
+                    </ul>
+                        
+                </nav>
+                <section class="hero-section">
+                    <div class="hero-content-wrapper">
+                        <div class="hero-content">
+                            <h1 class="hero-title">Organise projects.<br>
+                                Get more done.
+                            </h1>
+                            <button class="hero-button">Get Started</button>
+                        </div>
+
+                        <div class="hero-image-container">
+                            <img src="images/iphone.png" alt="hero image">
+                        </div>
+                    </div>
+                    <div class="hero-company-logos">
+                        <img src="images/logos/airbnb-logo.png" alt="Airbnb logo">
+                        <img src="images/logos/hubsport-logo.png" alt="Hubsport logo">
+                        <img src="images/logos/google-logo.png" alt="Google logo">
+                        <img src="images/logos/microsoft-logo.png" alt="Microsoft logo">
+                        <img src="images/logos/walmart-logo.png" alt="Walmart logo">
+                        <img src="images/logos/fedex-logo.png" alt="FedEx logo">
+                    </div>
+                </section>
+                
+                <div class="feature-section">
+                    <h1 class="feature-heading">Tailor-made features</h1>
+                    <p class="feature-p">Lorem ipsum is common placeholder text used to demonstrate the graphic elements of a document or visual presentation.</p>
+                </div>
+                <section class="feature-content">
```

Comment:
- [ ] Here, you can use `section` tag for each section and this `feature-section` should be a `section` as well
- [ ] And `feature-content` can be a normal `div`.
- [ ] `feature-content` should be inside the `feature-section`